dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -274,7 +274,12 @@ class CompileTest {
         watch.start();
         await flutter('build', options: options);
         watch.stop();
-        releaseSizeInBytes = await file('$cwd/build/app/outputs/apk/app-release.apk').length();
+        File apk = file('$cwd/build/app/outputs/apk/release/app-release.apk');
```

Comment:
This works. We could also just take the copy at `build/app/outputs/apk/app.apk` like the Flutter tools do?

Reply:
Hmm, will check this option too.